files/build_templates/qos_config.j2

```diff
@@ -77,6 +77,13 @@
 {% endif %}
 {%- endfor %}
 
+{%- set LOSSLESS_TC = [] %}
+{%- if generate_lossless_tc_list is defined %}
+    {{- generate_lossless_tc_list(LOSSLESS_TC) }}
+{%- else %}
```

Comment:
Thanks @alpeshspatel, this is good. If we need to have different set of LOSSLESS_TCs, we could use this macro inside SKUs.

Reply:
Yes Kumaresh - that is the intent